superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -432,6 +432,29 @@ export const controls = {
     'to find in the [country] column'),
   },
 
+  freq: {
+    type: 'SelectControl',
+    label: t('Frequency'),
+    default: 'W-MON',
+    freeForm: true,
+    clearable: false,
+    choices: [
+      ['AS', '[AS] Year'],
+      ['52W-MON', '[52W-MON] 52 Weeks (starting Monday)'],
```

Comment:
should there be an equivalent for `52 Weeks (starting Sunday)`?

Reply:
Seems reasonable to add in the default list. It's just hard to know where to stop (other days of the week?). My intent was to show a few example expression and let the users compose their own following the pattern. I'll clarify the help text and add more examples.

Reply:
Oh I see, yeah no need to add. I thought you were maybe doing the defaults based on known use-cases as opposed to generic variety so I think not adding is 👍